clang/test/OpenMP/depobj_codegen.cpp

```diff
@@ -17,6 +17,15 @@
 typedef void *omp_depend_t;
 
 void foo() {}
+void tmainc(){
+   omp_depend_t obj;
+#pragma omp depobj(obj) depend(inout: omp_all_memory)
+{
+   volatile omp_depend_t temp = obj;
+    char* char_ptr = reinterpret_cast<char*>(temp);
+    char_ptr[0] = 1;
+}
```

Comment:
even though we don't enforce code format in test, could you just do it manually?

```
#pragma omp depobj(obj) depend(inout: omp_all_memory)
  {
    ...
  }
```

Reply:
@shiltian  Is this correct for what you asked ?